client/src/components/History/CurrentHistory/HistoryPanel.vue

```diff
@@ -3,6 +3,7 @@
         :key="history.id"
         :historyId="historyId"
         :offset="offset"
+        :filterText="params.filterText"
```

Comment:
I'm curious:
My [eslint-plugin-vue](https://eslint.vuejs.org/) is telling me that [Attribute ':filterText' must be hyphenated.](https://eslint.vuejs.org/rules/attribute-hyphenation.html) Same for `historyId`, `showDeleted` and `showHidden`.

Should we follow this naming convention for consistency?

The thing is if I try to use the correct naming here then I get a bunch of `TypeError: Cannot read properties of undefined (reading 'length')` errors in the console I think because the HistoryItemsProvider is expecting the camel case properties. 

Do you know why it doesn't work in this case? I'm still a bit lost with Vue magic...

Reply:
Yes, its because we are parsing the attributes directly to the stores and the attribute keys are not converted to camel case. I added a method to the generic store provider to make sure that the attached stores get consistent attribute keys.

Reply:
Nice! thank you!